static/bidder-info/ix.yaml

```diff
@@ -18,4 +18,6 @@ userSync:
   redirect:
     url: "https://ssum.casalemedia.com/usermatchredir?s=194962&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
     userMacro: ""
+  iframe:
+    url: "https://js-sec.indexww.com/um/ixmatch.html"
```

Comment:
This doesn't look like a valid user match url. How does the iframe code know where (url of PBS) to redirect users? Are privacy policies supported?

Reply:
Thanks @SyntaxNode. Updated